src/styles/main.css

```diff
@@ -0,0 +1,126 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Montserrat:wght@100;700&display=swap');
+
+:root {
+  --pale-gray: #f5f6fa;
+  --azure: #0290ff;
+  --white: #e8e8e8;
+  --medium-blue: #307bbe;
+  --dodger-blue: #379cf6;
+}
+
+.Text-Style-5 {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.flex {
+  display: flex;
+}
+
+.panel-bg {
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+}
+
+.Bookstore-CMS {
+  width: 100rem;
+  height: 89.625rem;
+  margin: 0 auto;
+  padding: 9.313rem 3.75rem 7.688rem 6.25rem;
+  background-color: var(--pale-grey);
+}
```

Comment:
Your style looks almost identical to the one shown in the Zeplin design.

However, one of the project requirements was Your final design should match Zeplin's design.
Therefore, we must stick to the requirement. Currently, your Bookstore app is missing pale grey background around your main app.

- [ ] Please add the background as in the Zeplin;
- [ ] You have a horizontal overflow, please remove it;

Here is how it looks like as per the Zeplin (notice there is grey background around main app window).
<img width="1440" alt="Screen Shot 2021-11-05 at 23 47 00" src="https://user-images.githubusercontent.com/60043698/140572479-9433f5b7-9b47-4ce9-8a4a-c811b42875a2.png">

Reply:
Thanks for the guide. I have refactor the design with the background added.